pegasus/sites.v3/code.org/public/student/middle-high.haml

```diff
@@ -1,185 +1,196 @@
 
-  .action-block__wrapper.action-block__wrapper--three-col
-    .action-block.action-block--one-col.flex-space-between
-      .content-wrapper
-        %p.overline
-          grades
-          =hoc_s(:module_grade_6_12)
-        %h3
-          =hoc_s(:middle_high_page_cs_card_title_1)
-        %img{src: "/images/courses-catalog/CSD.png", alt: ""}
-        %p
-          =hoc_s(:middle_high_page_cs_card_description_1)
-      .content-footer
-        %a.link-button{href: "https://code.org/csd", target: "_blank", rel: "noopener noreferrer"}
-          =hoc_s(:middle_high_page_cs_link_1)
-    .action-block.action-block--one-col.flex-space-between
-      .content-wrapper
-        %p.overline
-          grades 
-          =hoc_s(:module_grade_9_12)
-        %h3
-          =hoc_s(:middle_high_page_cs_card_title_2)
-        %img{src: "/images/courses-catalog/CSP.png", alt: ""}
-        %p
-          =hoc_s(:middle_high_page_cs_card_description_2)
-      .content-footer
-        %a.link-button{href: "https://code.org/csp", target: "_blank", rel: "noopener noreferrer"}
-          =hoc_s(:middle_high_page_cs_link_2)
-    .action-block.action-block--one-col.flex-space-between
-      .content-wrapper
-        %p.overline
-          grades
-          =hoc_s(:module_grade_9_12)
-        %h3
-          =hoc_s(:middle_high_page_cs_card_title_3)
-        %img{src: "/images/courses-catalog/CSA.png", alt: ""}
-        %p
-          =hoc_s(:middle_high_page_cs_card_description_3)
-      .content-footer
-        %a.link-button{href: "https://code.org/csa", target: "_blank", rel: "noopener noreferrer"}
-          =hoc_s(:middle_high_page_cs_link_3)
+    .action-block__wrapper.action-block__wrapper--three-col
+      .action-block.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline
+            grades
+            =hoc_s(:module_grade_6_12)
+          %h3
+            =hoc_s(:middle_high_page_cs_card_title_1)
+          %img{src: "/images/courses-catalog/CSD.png", alt: ""}
+          %p
+            =hoc_s(:middle_high_page_cs_card_description_1)
+        .content-footer
+          %a.link-button{href: "https://code.org/csd", target: "_blank", rel: "noopener noreferrer"}
+            =hoc_s(:middle_high_page_cs_link_1)
+      .action-block.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline
+            grades
+            =hoc_s(:module_grade_9_12)
+          %h3
+            =hoc_s(:middle_high_page_cs_card_title_2)
+          %img{src: "/images/courses-catalog/CSP.png", alt: ""}
+          %p
+            =hoc_s(:middle_high_page_cs_card_description_2)
+        .content-footer
+          %a.link-button{href: "https://code.org/csp", target: "_blank", rel: "noopener noreferrer"}
+            =hoc_s(:middle_high_page_cs_link_2)
+      .action-block.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline
+            grades
```

Comment:
Would it be possible to use a translatable string for "grades" in the few places it's explicitly typed out in this file?

Reply:
Ah yes, definitely! I missed this when reviewing Dmytro's original PR 🤦‍♀️